Tools/Scripts/webkitdirs.pm

```diff
@@ -619,14 +617,19 @@ sub determineXcodeDestination
     }
         
     if (!$generic && $xcodeSDKPlatformName =~ /simulator$/) {
-        my $runtime = iosSimulatorRuntime();
+        my $runtime = simulatorRuntime($portName);
```

Comment:
This works because the port names for simulated platforms (iOS, tvOS, visionOS, watchOS) match the platform names used by the simulator.